marlowe-playground-client/package.json

```diff
@@ -35,7 +35,7 @@
     "file-loader": "^2.0.0",
     "html-webpack-plugin": "^3.2.0",
     "monaco-editor-webpack-plugin": "^1.9.0",
-    "nearley-webpack-loader": "https://github.com/shmish111/nearley-webpack-loader/archive/master.tar.gz",
+    "nearley-webpack-loader": "github:shmish111/nearley-webpack-loader#939360f9d1bafa9019b6ff8739495c6c9101c4a1",
```

Comment:
i had to change this because i *need* a specific commit for the fetcher in npmlock2nix. On top of that we should probably just do that anyway since we want reproducible builds.